src/storage/src/monitor/mod.rs

```diff
@@ -23,3 +23,28 @@ pub use hummock_metrics::*;
 mod local_metrics;
 pub use local_metrics::StoreLocalStatistic;
 pub use risingwave_object_store::object::object_metrics::ObjectStoreMetrics;
+
+#[cfg(hm_trace)]
+mod traced_store;
+use futures::Future;
+#[cfg(hm_trace)]
+pub use traced_store::*;
+#[cfg(all(not(madsim), hm_trace))]
+use {
+    risingwave_hummock_trace::{ConcurrentId, CONCURRENT_ID, LOCAL_ID},
```

Comment:
We don't need to export so many detailed fields from `risingwave_hummock_trace`. We only need to export a function named `trace_hummock`, can call it in the `may_trace_hummock` below under `#[cfg(hm_trace)]`